api/log/log_test.go

```diff
@@ -66,7 +66,7 @@ func TestLog(t *testing.T) {
 		},
 		{
 			name:         "Testing log.Error fail",
-			logger:      &stubLogger{err: errors.New("server is down!")},
+			logger:       &stubLogger{err: errors.New("server is down!")},
```

Comment:
It looks like your lint has added this space.

Reply:
Yeah, i think it was trying to align all variables inside the struct 😅